tests/smoke-tests/test-staking-rewards.ts

```diff
@@ -278,8 +295,8 @@ async function assertRewardedEventsAtBlock(
 
     if (apiAtBlock.events.parachainStaking.Rewarded.is(event)) {
       rewardCount++;
-      rewards[event.data[0].toHex().toUpperCase()] = {
-        account: event.data[0].toHex().toUpperCase(),
+      rewards[toUniform(event.data[0])] = {
```

Comment:
Why not use the toHex instead of creating a new function toUniform?

Reply:
For some reason, the values from the storage and events were inconsistent in case when only `toHex()` or `toString()` was used.

Reply:
Yes, toHex() != toString()
toHex() => `lowerCase`
toString() => `cHecKsum cAse`

If you stay consistent within your code you shouldn't need a formatter